tools/check.py

```
@@ -0,0 +1,248 @@
+
+cpp = cpp_lib()
+
+
+cells = (200, 100)
+time_step = 0.005
+final_time = .01
+timestamps = [0, final_time/2,final_time]
+
+def _run(s):
+    args = s.split(" ")
+    result = subprocess.run(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, universal_newlines=True)
```

Ruff (0.14.8)

`subprocess` call: check for execution of untrusted input

(S603)